### ISSUE TITLE
Stage/Use yt-dlp from ubuntu repos

### ISSUE THREAD

**Issue by sameersharma2006** [sameersharma2006/xdman-snap#2]
Xdm seems like a dead project, the last release was in 2023, hence its code may not be compatible with latest yt-dlp releases , hence it will be ideal to use yt-dlp from ubuntu repos(currently xdm snap is built atop jammy).
Xdm builds on ubuntu 22.04,hence its yt-dlp will be much better supported than the newer ones .

**Comment by dnet890** [comment]
yeah, on Ubuntu 24.04. I cannot close the app properly. Is it a bug?
